com.vysp3r.ProtonPlus.json

```diff
@@ -0,0 +1,24 @@
+{
+    "app-id" : "com.vysp3r.ProtonPlus",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "protonplus",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--talk-name=org.gtk.vfs.*",
+        "--filesystem=xdg-run/gvfsd"
+    ],
+    "modules" : [
+        {
+            "name" : "ProtonPlus",
+            "builddir" : true,
+            "buildsystem" : "meson"
```

Comment:
this is no source. I wonder how it is supposed to find it.

Reply:
I will ask someone on the forum on how that part work, I did not really understand this.

Reply:
I think I've fixed it